### PR TITLE
Fix blackslake margin

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: scss/components/blankslate.scss [hunk 1]
```diff
@@ -1,21 +1,19 @@
 .blankslate {
-  padding: 30px;
-  margin: 20px;
+  padding: $blackslate-padding;
```
Comment (GermainBergeron) [inline scss/components/blankslate.scss]: should be `blankslate-padding`

File: scss/components/blankslate.scss [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 
   h1, h2, h3, h4, h5, h6 {
-    margin-top: 15px;
-    margin-bottom: 15px;
+    margin: blackslate-margin 0 blackslate-margin 0;
   }
 
   p {
-    margin-bottom: 15px;
+    margin-bottom: blackslate-margin;
```
Comment (GermainBergeron) [inline scss/components/blankslate.scss]: `blankslate-margin`

**Comment by GermainBergeron** [comment]
:shipit:

**Comment by JPLachance** [comment]
:ship: :it: !

**Pull request merged**